### PR TITLE
New package: BeagleSoftware.BeTL version 0.1.0

### PR DESCRIPTION
**Pull request by ManiProjs**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/188136)

### CODE REVIEW

**Comment by ManiProjs** [comment]
Can anyone approve this PR?

**Comment by ManiProjs** [comment]
I thought someone can answer to my question?

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Programs\BeagleEditor Toolchain (BeTL)\betl.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 897.)

**Comment by ManiProjs** [comment]
It caused because validator not have Microsoft.VCRedist.2015… installed. I
think it works on your own PC

File: manifests/b/BeagleSoftware/BeTL/0.1.0/BeagleSoftware.BeTL.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: BeagleSoftware.BeTL
+PackageVersion: 0.1.0
+InstallerType: inno
+Installers:
```
Comment (stephengillie) [inline manifests/b/BeagleSoftware/BeTL/0.1.0/BeagleSoftware.BeTL.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
image

File: manifests/b/BeagleSoftware/BeTL/0.1.0/BeagleSoftware.BeTL.installer.yaml (hunk 1 above)
Comment (ManiProjs) [reply manifests/b/BeagleSoftware/BeTL/0.1.0/BeagleSoftware.BeTL.installer.yaml:7]: Thank you @stephengillie. I will accept it

**Comment by ManiProjs** [comment]
@stephengillie Attention! Please approve this PR, please!

**Comment by stephengillie** [comment]
Verify:
image
I couldn't get the alias to work, but that might be an issue with my VMs.

**Review by stephengillie — APPROVED** [review]

**Comment by ManiProjs** [comment]
> I couldn't get the alias to work, but that might be an issue with my VMs.

I don't think so. Installer does not add executable to PATH automatically; I wanted to do that, but my environment variable ruined. I fixed it with a lot of misery.

**Pull request merged**